src/index.ts

```diff
@@ -439,6 +438,23 @@ class IB extends EventEmitter {
     return this;
   };
 
+  reqPnLSingle = function (reqId, account, modelCode, conId) {
```

Comment:
can we use typescript types instead, I've been trying to get rid of all the `assert` lines

Reply:
Added typescript notation to the newly added / modified functions, however I would stick with the asserts and keep it. Types can still be wrong, especially when called from JS ;)